tests/unit/test_create_movie.py

```diff
@@ -1,7 +1,10 @@
 # TODO: Feature 2
 from src.models.movie import Movie
+from src.repositories.movie_repository import movie_repository_singleton
 
 def test_create_movie():
+    
+    movie_repository_singleton.create_movie('Forrest Gump', 'Robert Zemeckis', 4)
```

Comment:
This doesn't actually test anything, you create the movie, yes, but then you don't test that the movie was actually added to the DB. Try to get the movie from the `movie_repository_singleton` instance by name using another one of its methods and then assert that you were able to get the movie out after creating it.